src/Resources/views/Admin/GiftCard/Create/_javascripts.html.twig

```diff
@@ -0,0 +1,6 @@
+<script src="{{ asset('bundles/setonosyliusgiftcardplugin/setono-sylius-gift-card-send-notification-email.js') }}"></script>
+<script>
+    $(() => {
```

Comment:
Can you check if this worked?

I know there is a shorthand. You can read about it here: https://learn.jquery.com/using-jquery-core/document-ready/

Reply:
Yes it does